### PR TITLE
organization modal

### PR DESCRIPTION
**Pull request by rkzel**
This is for just displaying the modal. Not very functional by now, it's mainly missing storing the address somewhere: whihc means validation is also complaining. It's ready for integration though. Also, Organizations Card uses Education tiles until it has its own tiles (created in another ticket).

### CODE REVIEW

File: src/client/components/OrganizationPanel.js
Comment (Schwartz10) [inline src/client/components/OrganizationPanel.js]: If you're going to reuse this component, please change the name of the component, so it makes more sense.

File: src/client/components/modals/Organization.js
Comment (Schwartz10) [inline src/client/components/modals/Organization.js:39]: This is breaking the modal from opening, because there are no `organizations` in forms in state (you did a great job adding the `organizations` to the `publicProfile` in state.

I think you're missing `organizations` [here](https://github.com/AutarkLabs/aragon-profile/blob/master/src/client/stateManagers/box/states.js#L38)

File: src/client/components/modals/Organization.js
Comment (Schwartz10) [inline src/client/components/modals/Organization.js]: Not sure if this would work, it might need to be a raw eth address

File: src/client/utils/validation.js
Comment (Schwartz10) [inline src/client/utils/validation.js:16]: Does this work? Does that actually parse strings and expect an ethereum address?

File: src/client/components/OrganizationPanel.js
Comment (Schwartz10) [inline src/client/components/OrganizationPanel.js:16]: I think this is _okay_ for now, but it would be nice to remove any unnecessary variables if possible. If we could just add these as props that would save us another variable here. But i think this pattern exists elsewhere in the app, so it might be easier to just make one sweeping change. What are your thoughts?

**Review by Schwartz10 — CHANGES_REQUESTED** [review]
Hey, this is really close. You just need to update the `forms` in state to add room for `organizations`. Will wait for @chadoh to review as well before merging.

File: src/client/utils/validation.js
Comment (rkzel) [reply src/client/utils/validation.js:16]: It works, because:

validator.addFormat('address', {
  type: 'string',
  validate: isAddress,
})

and isAddress is the one from web3.

File: src/client/components/OrganizationPanel.js
Comment (rkzel) [reply src/client/components/OrganizationPanel.js]: I wasn't going to leave it, it was to be until `Render single organization info (logic) #8` is completed. However, it makes sense: I created a basic component on which #8 will be based.

File: src/client/components/modals/Organization.js
Comment (rkzel) [reply src/client/components/modals/Organization.js:39]: Yes, thx.

File: src/client/components/OrganizationPanel.js
Comment (rkzel) [reply src/client/components/OrganizationPanel.js:16]: if only "Object.keys(organizations).length" could be expressed in more concise way... How about replacing it with _.isEmpty() from lodash? Remove extra variable, replace the length test with isEmpty?

File: src/client/components/modals/Organization.js
Comment (rkzel) [reply src/client/components/modals/Organization.js]: Unfortunately it needs to be 0x... Fixed.

File: src/client/components/OrganizationPanel.js
Comment (Schwartz10) [reply src/client/components/OrganizationPanel.js:16]: I was referring to `cardProps`, not `organizationNotEmpty`. But i don't think we should add another dependency just to make that check. The check you're making is fine and can still be written in one line anyways.

**Comment by Schwartz10** [comment]
@chadoh we're going to close this for #29 and work off one commit

**Pull request closed without merging**
